server/tserver/src/main/java/org/apache/accumulo/tserver/scan/NextBatchTask.java

```diff
@@ -89,14 +88,16 @@ public void run() {
       }
     } catch (TooManyFilesException | SampleNotPresentException e) {
       addResult(e);
-    } catch (OutOfMemoryError ome) {
-      Halt.halt("Ran out of memory scanning " + scanSession.extent + " for " + scanSession.client,
-          1);
-      addResult(ome);
-    } catch (Throwable e) {
+    } catch (Exception e) {
       log.warn("exception while scanning tablet "
           + (scanSession == null ? "(unknown)" : scanSession.extent), e);
       addResult(e);
+    } catch (Error t) {
+      log.warn(
+          "Error while scanning tablet " + (scanSession == null ? "(unknown)" : scanSession.extent),
+          t);
```

Comment:
The message here has changed and no longer has the scan session client included. Also, you should use slf4j substitution syntax to format the message.

Are we sure we want to change the behavior to no longer Halt here? Is this change dependent on the one in #1818 to handle the shutdown?

Reply:
> The message here has changed and no longer has the scan session client included.
I'm not sure what yo umean, this is an copy of the log message in the catch clause above it.

> Also, you should use slf4j substitution syntax to format the message.
I can do that

> Is this change dependent on the one in #1818 to handle the shutdown?
Yes, I mentioned that in the initial comment on the PR. These changes assume that #1818 gets merged.

Reply:
> I'm not sure what yo umean, this is an copy of the log message in the catch clause above it.

The previous message from the Halt had `" for " + scanSession.client`, but this log message does not mention the `scanSession.client` at all.

> Yes, I mentioned that in the initial comment on the PR. These changes assume that #1818 gets merged.

Thanks, I must've missed that.

Reply:
As the log message you are referencing is in an Error handler, I should remove it as well given your comments in LookupTask?

Reply:
probably :smiley_cat: